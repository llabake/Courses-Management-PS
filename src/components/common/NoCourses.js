import React from 'react';
import PropTypes from 'prop-types';

const NoCourses = ({ text }) => (
  <div className="text-center">
    <h4>
      {text}
    </h4>
  </div>
);

NoCourses.propTypes = {
  text: PropTypes.string.isRequired,
};

export default NoCourses;
