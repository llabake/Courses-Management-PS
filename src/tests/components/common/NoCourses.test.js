import React from 'react';
import { shallow } from 'enzyme';

import NoCourses from "../../../components/common/NoCourses";

const wrapper = shallow(<NoCourses  text={'what'}/>);

test('<NoCourses /> mounts correctly', () => {
  expect(wrapper).toMatchSnapshot();
});

