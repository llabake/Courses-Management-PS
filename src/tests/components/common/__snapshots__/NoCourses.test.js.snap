// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoCourses /> mounts correctly 1`] = `
<div
  className="text-center"
>
  <h4>
    what
  </h4>
</div>
`;
